### DISCUSSION TITLE
About the Popover/Tooltip placement property

### DISCUSSION THREAD

**Discussion by ylazy** [adobe/react-spectrum#3286] [Q&A] [answered]
> A popover is positioned in relation to its source. The placement property values are the following: **top, top left, top right, top start, top end, bottom, bottom left, bottom right, bottom start, bottom end, left, left top, left bottom, start, start top, start bottom, right, right top, right bottom, end, end top, end bottom.** The default placement value is at the top.
(https://spectrum.adobe.com/page/popover/#Placement)

> A tooltip is positioned in relation to its target. Placement property values are at the: **top, top left, top right, top start, top end, bottom, bottom left, bottom right, bottom start, bottom end, left, left top, left bottom, start, start top, start bottom, right, right top, right bottom, end, end top, end bottom**. The default placement value is at the top.
(https://spectrum.adobe.com/page/tooltip/#Placement)

I can't understand these placements:
```
top left, top right (vs. top start, top end)
bottom left, bottom right (vs. bottom start, bottom end)
start, start top, start bottom
end, end top, end bottom
```
I can't find any description of these placements on Spectrum, Spectrum CSS, React Spectrum, or XD files.
Any ideas are welcome!
Thanks!

**Comment by reidbarber** [comment, marked as answer]
Start and end placements are dependent on language direction.

So for left-to-right languages like English, start will be the same as left.

And for right-to-left languages like Arabic, start will be the same as right.

It's generally better to use start and end in order to match the flow of the document and have better support for internationalization.

**Reply by ylazy** [reply]
Thank you so much!
